src/components/Button/Button.scss

```diff
@@ -56,6 +56,25 @@
   background-color: $grey;
 }
 
+.button--filter {
+  background-color: $white;
+  color: $black;
+  border: 0.2rem solid $black;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  min-width: 9rem;
+  margin-right: 1rem;
+  margin-left: 1rem;
+
+  &--active {
```

Comment:
should this be a pseudoclass? `&:active` ? otherwise we have a modifier on a modifier!

Reply:
you caught me red handed! i dont think a  pseudoclass provides us with a toggled button state, :active is limited to only when button is being clicked